### PR TITLE
PF-1489 - Adding selfie app example

### PR DESCRIPTION
**Pull request by fredcido**
Adding app example to take user's selfie and allow updating the image with custom actions.

### CODE REVIEW

File: examples/selfie/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+## Self Image
```
Comment (anthonyroux) [inline examples/selfie/README.md:1]: Hey @fredcido, we have introduced a new template for the readme.md for all our public demo examples based on the feedback we received from developers.

You can see one [here](https://github.com/miroapp/app-examples/tree/main/examples/drag-and-drop) for example.

You can find the template.md [here](https://github.com/miroapp/app-examples/blob/main/README.template.md)

Cc: @horeaporutiu

Comment (fredcido) [reply examples/selfie/README.md]: Thanks @anthonyroux , I know the new template, I am preparing the PR for review, it's still in draft.

File: examples/selfie/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+3. Open the [app manifest editor](https://developers.miro.com/docs/manually-create-an-app#step-2-configure-your-app-in-miro) by clicking **Edit in Manifest**. \
+   In the app manifest editor, configure the app as follows:
+
+   - [`sdkUri`](https://developers.miro.com/docs/app-manifest#sdkuri): assign `http://localhost:3000` as a value for this property. \
+     It defines the entry point of the app, and it corresponds to the URL of the server that the app runs on.
+   - [`scopes`](https://developers.miro.com/docs/app-manifest#scopes): add the permission scopes that users need to grant the app when they install it. \
+     To enable the app to read from and write to the board, add the following permissions:
+     - `boards:read`
+     - `boards:write`
+     - `webcam:record`
```
Comment (mettin) [inline examples/selfie/README.md:64]: Can you provide the yaml snippet here?

File: examples/selfie/package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "self-image",
+  "version": "0.1.0",
+  "license": "MIT",
+  "description": "This example shows you how to take selfies in Miro and update them using custom actions.",
```
Comment (mettin) [inline examples/selfie/package.json:5]: Please add [keywords](https://github.com/miroapp/app-examples/blob/main/examples/calendar/package.json#L5)

File: examples/selfie/src/app.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import type { Image } from "@mirohq/websdk-types";
+import { createRoot } from "react-dom/client";
+
+const App: React.FC = () => {
+  const videoRef = React.useRef<HTMLVideoElement>(null);
+  const [state, setState] = React.useState<"idle" | "busy">("idle");
```
Comment (mettin) [inline examples/selfie/src/app.tsx:7]: Maybe its easier to read if you just use `isBusy` as a boolean
```ts
const [isBusy, setIsBusy] = useState(false)

File: examples/selfie/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
+import type { CustomAction, CustomEvent } from "@mirohq/websdk-types";
+
+const registerAction = async () => {
```
Comment (mettin) [inline examples/selfie/src/index.ts:3]: Can you move this function into its own file (i.e. `register-custom-miro-action.ts`)? That will make it easier to find the custom action creation for users

**Review by mettin — APPROVED** [review]
Can we change the example's folder name to include `custom-action`?

File: examples/selfie/README.md (hunk 2 above)
Comment (mettin) [reply examples/selfie/README.md]: [Yes](https://github.com/miroapp/app-examples/tree/main/examples/ai-image-generator#%EF%B8%8F-run-the-app-locally-)

**Review by mettin — APPROVED** [review]

**Pull request merged**